### ISSUE TITLE
404 - The requested URL timed out

### ISSUE THREAD

**Issue by aletuba** [andrieslouw/imagesweserv#302]
Probably my server is blocking the weserv IP. Can you tell me the IP to check?

**Comment by kleisauke** [comment]
A complete list of the IPs we use for outbound traffic can be found here: https://images.weserv.nl/ips.txt

**Comment by kleisauke** [comment]
I hope this information helped. Please feel free to re-open if questions remain.

**Comment by abubelinha** [comment]
@kleisauke the title of the issue fits my problem, but I don't know if you prefer me to open a new issue 
(I guess the answers to a 404 problem maybe specific to each particular case).

I am generating a local HTML file wich contains thumbnails of about 5000 images served by my institution [iipserver](https://iipimage.sourceforge.io/documentation/server).
(I want them to print this HTML to a PDF, so I generate a yearly index of all the images in our museum collection).

If I just load my html file without modifying our urls, the images take a long time to be loaded ... but they finally appear (all of them). As this is slow and the PDF is bigger than I want, I tried to use your cache service to reduce the load time and also to reduce the thumbnails size.

I used `https://wsrv.nl/?url=` as a prefix, then I put the encoded url of the thumbnail, and finally I added `&w=50` 
(our thumbnails are about 100 px wide)

If I try to load my html file with the modfied urls, I believe wsrv.nl is trying to fetch all these images faster than our iipserver is able to serve them.  So I only get 404 answers.
Is there any way that I can increase the time wsrv.nl waits for them to be served?

I know my wsrv.nl-urls are OK: 
if I load them one by one in my browser, the images are fetched and cached without any problems.
(as long as I didn't try to use them already from my html file: in that case, the 404 json response is already cached and that's what I get from the browser the next time I try to load the image, indeed).

So, these are my questions:
- Is there any weserv url parameter that I can add to increase the timeout before wsrv.nl decides to return 404?
- Do I need to create a script that tries to slowly load all the images using weserv on by one ... before trying to load them all together from my html index file?  
- Is there any weserv url parameter that I can add to recache the same image again? (so I can use the "slow script" without having to modify the w parameter)
- How long do 404.json messages get cached by wsrv.nl?  (maybe I can try again later without having to modify my w parameter at all)

Thanks a lot!
@abubelinha
